### ISSUE TITLE
Root query and mutation don't have __typename

### ISSUE THREAD

**Issue by svrcekmichal** [Youshido/GraphQL#88]
Query:
```
mutation { // or query or empty
  __typename
}
```
Response: 
```
{
  "errors": [
    {
      "message": "Field \"__typename\" not found in type \"Mutation\"",
      "locations": [
        {
          "line": 20,
          "column": 3
        }
      ]
    }
  ]
}
```

BTW: Big thanks for locations key 👍

**Comment by portey** [comment]
Fixed and waiting for new release.

**Comment by viniychuk** [comment]
Published in v1.4.2.6
